tests/units/checkTokenMiddleware.test.js

```diff
@@ -0,0 +1,31 @@
+const {
+  default: checkTokenMiddleware,
+} = require("@/middlewares/checkTokenMiddleware");
+
+describe("checkTokenMiddleware", () => {
+  const res = { status: jest.fn().mockReturnThis(), json: jest.fn() };
+  const req = { headers: { token: undefined } };
+  it("Should respond with code 400 if the provided token is invalid", async () => {
+    req.headers.token = process.env.INVALID_TOKEN;
+    await checkTokenMiddleware(req, res);
+    expect(res.status).toBeCalledWith(400);
+    expect(res.json).toBeCalledWith({
+      status: false,
+      message: "Invalid token",
+    });
+  });
+  it("Should respond with code 400 if there is no user related to the provided token ", async () => {
+    req.headers.token = process.env.TOKEN_NOT_EXIST;
```

Comment:
why process.env? just give it a sample data

Reply:
The tokens may be needed in other unit and integration tests. By putting it in the .env, our code will be cleaner without redundancy.